light-system-programs/tests/functional_tests.ts

```diff
@@ -84,6 +79,95 @@ describe("verifier_program", () => {
     });
   });
 
+  it("Deposit (verifier one)", async () => {
+    await performDeposit({
+      delegate: AUTHORITY_ONE,
+      spl: true,
+      senderSpl: userTokenAccount,
+      shuffleEnabled: true,
+      verifierIdl: IDL_VERIFIER_PROGRAM_ONE,
+    });
+  });
+
+  it("Deposit (verifier storage)", async () => {
+    await performDeposit({
+      delegate: AUTHORITY,
+      spl: false,
+      message: Buffer.alloc(900).fill(1),
+      senderSpl: null,
+      shuffleEnabled: false,
+      verifierIdl: IDL_VERIFIER_PROGRAM_STORAGE,
+    });
+  });
+
+  it("Deposit (verifier zero)", async () => {
+    await performDeposit({
+      delegate: AUTHORITY,
+      spl: true,
+      senderSpl: userTokenAccount,
+      shuffleEnabled: true,
+      verifierIdl: IDL_VERIFIER_PROGRAM_ZERO,
+    });
+    const lightProvider = await Provider.init({
+      wallet: ADMIN_AUTH_KEYPAIR,
+      relayer: RELAYER,
+      confirmConfig,
+    });
+    await lightProvider.relayer.updateMerkleTree(lightProvider);
```

Comment:
what do you think about adding an option, updateMerkleTree to performDeposit ?
```
await performDeposit({
      delegate: AUTHORITY,
      spl: true,
      senderSpl: userTokenAccount,
      shuffleEnabled: true,
      verifierIdl: IDL_VERIFIER_PROGRAM_ZERO,
     updateMerkleTree: true
    });
 ```

Reply:
Good point, added.